### ISSUE TITLE
When an anchor target also moves

### ISSUE THREAD

**Issue by kaifresh** [Prinzhorn/skrollr#671]
Im making a cliche parallax scroller a la: http://prinzhorn.github.io/skrollr/examples/classic.html but I'm having trouble using anchor targets. One of my translating divs uses a second translating div as its anchor target. However when the bottom of the anchor target hits the top of the screen, it doesn't trigger the translation. I had to use an offset to actually get it to fire. Perhaps I can show you better than I can tell you:
<em>THIS WORKS FINE</em>
   <section id="one-day-djs" 
                  data-anchor-target="#djs" 
                  data-top="transform:translate(0%,-100%);" 
                  data-top-bottom="transform:translate(0%,-200%);"
                 >

<em>THIS FEELS REALLY HACKY, BUT WORKS. </em>
         <section id="about" 
                  data-anchor-target="#one-day-djs"
                  data-bottom-top="transform:translate(0%, -196%); background-color:rgb(0,255,0)" 
                  data--110p-bottom=" transform:translate(0%, -350%); background-color:rgb(0,0,255)"
                 >
My question is: <i>do you have to use static anchor targets for it to work properly? </i> I feel like the reason the offset works is because it is using the initial position of the anchor target as the start point, and not the translated one (i.e. one whose positions keeps changing). <b>Is this true? And is there a neater way to do this?</b>

Secondly, you can see that I'm creating a vertically stacked set of divs and then repositioning them just as they need to come on screen. Is there a better initial layout? I notice in the example (from the url above) that you have the images all at the top at the start (i.e. out of order) and then move them down into the correct position just as the screen hits them. Alternatively is it just like, if it works, then do that?

**Comment by Prinzhorn** [comment]
> do you have to use static anchor targets for it to work properly?

yes. It's impossible to predict/know where the anchor is if it is animated instead of simply following the natural flow.

Regarding the rest of your question: I have problems following it. Can you upload/create an example?
